contracts/mocks/SpotOracleMock.sol

```diff
@@ -9,6 +9,6 @@ contract SpotOracleMock is ISpotOracle {
     constructor () { }
 
     function setSpot(uint256 spot_) public {
-        spot = spot_;
+        spot = spot_ * 1e27;
```

Comment:
This was actually right, although a comment from my part would have been useful. `spot` must be given as a 27 decimal number, so a `spot` of 2000 would actually be 2000 * 10^27, and a `spot` of 0.0005 would be 5 * 10^23.

Posting and then testing `powerOf` would show this is right. If we set `spot` to 2000 * 10^27 and `post` 1 ETH, `powerOf` should return 2000 * 10^18 (or 2000 Dai).

Reply:
I know! But it proved impossible to set spot to 2000^1e27, because the JS library can't handle so big a big number.

Reply:
Did you try this?

```
const RAY = BigNumber.from(10).pow(27)
const SPOT = RAY.mul(2000)
```

Reply:
Hmm... It works! Tried something similar initially, which set me down... Alright, you're right!